### PR TITLE
Dedup columns when converting to lower case

### PR DESCRIPTION
**Pull request by sean-zou**

### CODE REVIEW

File: src/csv_processor/lambda.py [hunk 1]
```diff
@@ -7,7 +7,43 @@
...
+            if sensitive_val.islower():
+                update_substr = update_substr +'_lower'
+            elif sensitive_val.isupper():
+                update_substr = update_substr+'_upper'
+            else :
+                update_substr = update_substr+'_camel'
+            old_column_val_updated = old_split[0]+':'+update_substr
+        lower_case_val = re.sub('([A-Z]+)', r'_\1',old_column_val_updated).lower()
+        lower_case_val = re.sub('[^0-9a-zA-Z]+', '_', lower_case_val)
+        duplicate_check = duplicate_check_dict.get(lower_case_val)
```
Comment (tosih) [inline src/csv_processor/lambda.py:31]: Can you do something like this to dedup? `mylist = list(dict.fromkeys(mylist))`

File: src/csv_processor/lambda.py [hunk 2]
```diff
@@ -7,7 +7,43 @@
 import pyarrow.parquet as pq
 import pyarrow as pa
 import s3fs
+import re
 
+def dedup_columns(oldColumns):
```
Comment (tosih) [inline src/csv_processor/lambda.py:12]:
```suggestion
def dedup_columns(old_columns):
```

File: src/csv_processor/lambda.py [hunk 3]
```diff
@@ -7,7 +7,43 @@
...
+    old_new_mapping = {}
+    duplicate_check_dict = {}
+
+    for old_column_val in oldColumns:
```
Comment (tosih) [inline src/csv_processor/lambda.py:16]:
```suggestion
    for old_column_val in old_columns:
```

File: src/csv_processor/lambda.py (hunk 1 above)
Comment (sean-zou) [reply src/csv_processor/lambda.py:31]: This is not the usual dedup. Wanna convert the column names from case sensitive to case insensitive and make sure no duplicates

**Pull request merged**